### PR TITLE
Feature/indicator dataset

### PR DESCRIPTION
**Pull request by adieyal**
## Description

Working on a feature to change the way that groups are handled, flagging them as filterable and aggregateable 

## Checklist

- [ ]  🚀 is the code ready to be merged and go live?
- [ ]  🛠 does it work (build) locally

### Pull Request

- [ ]  📰 good title
- [ ]  📝good description
- [ ]  🔖 issue linked
- [ ]  📖 changelog filled out

### Commits

- [ ]  commits are clean
- [ ]  commit messages are clean

### Code Quality

- [ ]  🚧 no commented out code
- [ ]  🖨 no unnecessary logging
- [ ]  🎱 no magic numbers
- [ ]  black was run locally (as part of the pre-commit hook)

### Testing

- [ ]  ✅ added (appropriate) unit tests
- [ ]  💢 edge cases in tests were considered
- [ ]  ✅ ran tests locally & are passing

### CODE REVIEW

File: tests/profile/test_views.py [hunk 1]
```diff
@@ -139,3 +130,27 @@ def test_matched_url(self, profile, rf):
...
+class TestProfileIndicatorData(
+    APIViewTest,
+    UsesGetMethod,
+    Returns200
+
+):
+
+    @pytest.fixture
+    def url(self, profile, geography, grouped_profile_indicator):
+        return reverse("profile-geography-indicator-data", kwargs={
```
Comment (milafrerichs) [inline tests/profile/test_views.py:145]: you don't need to use `reverse` here. 
we have `django-test-plus` which offers an `APITestCase` and you can just use `self.get(route, ...args)`
like here: 
https://github.com/OpenUpSA/wazimap-ng/blob/3a38bdfb5ea595d1220d0a2667224ec78a29b6cc/tests/profile/test_views.py#L62

Comment (adieyal) [reply tests/profile/test_views.py:145]: that's the standard idiom for pytest-drf

**Comment by adieyal** [comment]
No longer relevant
